proto/greptime/v1/row.proto

```diff
@@ -33,6 +33,9 @@ message ColumnSchema {
   SemanticType semantic_type = 3;
   // Extension for ColumnDataType.
   ColumnDataTypeExtension datatype_extension = 4;
+
+  // "index.type": "inverted" to indicate inverted index
+  map<string, string> options = 5;
```

Comment:
Question: Can we use a struct?

Reply:
Do you mean we try to use static struct and explicit pre-defined key, instead of the dynamic map?

Reply:
Yes

Reply:
good idea. @waynexia @fengjiachun what about this proposal?

Reply:
`struct` looks good to me